src/utils/getNeededDates.js

```diff
@@ -1,6 +1,6 @@
 const { events } = require("../models/Event");
 
-function getNeededDates (year, month) {
+function getNeededDates(year, month) {
```

Comment:
Do we use this function anywhere anymore? If so, what to expect to return the "getNeededDates" function?

Reply:
it will return the start date and the end date of a given year and month

Reply:
I don't use this function anywhere more